packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -368,6 +372,26 @@ class RenderListWheelViewport
     markNeedsPaint();
   }
 
+  /// {@template flutter.rendering.wheelList.offCenterOpacity}
+  /// The opacity value that will be applied to the wheel that appears below and
+  /// above the magnifier.
+  ///
+  /// The default value is 1.0, which will not change anything.
+  ///
+  /// Must be greater than or equal to 0, and less than or equal to 1.
+  /// {@endtemplate}
+  double get offCenterOpacity => _offCenterOpacity;
+  double _offCenterOpacity = 1.0;
+  set offCenterOpacity(double value) {
+    assert(value != null);
+    assert(value > 0);
```

Comment:
Should this be `value >= 0`?  Looking at your comment.

Reply:
ah forgot to change it. nice catch!